code/modules/trade/datums/trade_stations_presets/1-common/asterstradecapital.dm

```diff
@@ -133,6 +133,7 @@
 		)
 	)
 	offer_types = list(
+		/obj/structure/closet/crate = offer_data("crate return program", 150, 5),
```

Comment:
I haven't played Guild in a week but given how the system works (more than 1 item = item comes in a crate) I am 99% sure you can abuse this to generate money out of nothing.

Reply:
150 credits every ~10 minutes or so is nothing. And what's the minimum cost on something imported, 10 credits?
100 credits of profit will barely scratch at the stations influence.

Reply:
The reset on a station is 10 minutes, right? Or at least about there.

Reply:
Man doesn't want to apply wrench to crate or drag it to the smelter that's 20 tiles west. The absolute state of eris.

Reply:
Finally, 500 credits every 7 minutes. 
(I see what you mean, I'll get rid of it since they can use the smelter to kill crates anyway)